server/src/main/java/io/crate/types/DataTypeXContentExtension.java

```diff
@@ -69,6 +71,19 @@
                 b.value(timetz.getMicrosFromMidnight());
                 b.value(timetz.getSecondsFromUTC());
                 b.endArray();
+            }),
+            Map.entry(BitString.class, (b, v) -> {
+                // TODO: Go with different format?
+                // This format affects:
+                //  - storage
+                //      - _raw
+                //      - doc-values
+                //      - reverse index field
+                //  - http output
+                BitString bitString = (BitString) v;
```

Comment:
Tried some different approaches:

| DocValues            | value               | Avg size per doc | Ingest 50 | Ingest 95 | Query mean  |
| ---------            | ------              | ---------------- | --------- | --------- | ----------- |
| SortedDocValuesField | bistring bytes      | 1849             | 744       | 11956     | 5.560       |
| SortedDocValuesField | base64 bitset bytes | 506              | 314       | 776       | 2.457       |
| SortedDocValuesField | bitset bytes        | 388              | 378       | 675       | 2.640       |
| SortedDocValuesField | bitstring base64    | 2400             | 662       | 8215      | 9.526       |
| BinaryDocValuesField | bistring bytes      | Not tested       |           |           |             | 
| BinaryDocValuesField | base64 bitset bytes | 505              | 385       | 762       | 2.613       |
| BinaryDocValuesField | bitset bytes        | 385              | 395       | 661       | 4.943       |
| BinaryDocValuesField | bitstring base64    | 1560             | 607       | 8508      | 5.611       |


See https://gist.github.com/mfussenegger/45f4bb0ebccccfffa898c711d634e898 for details

My conclusion:

- Storing the bitSet bytes directly works well. 
- We need to distinguish between HTTP format and internal storage format.
- Unclear whether `SortedDocValuesField` or `BinaryDocValuesField` works better. My assumption is that the former might work better if the data is biased. Should follow up here with some more tests. In my tests the bitstrings were randomized.